config/prow/plugins.yaml

```diff
@@ -831,6 +831,7 @@ plugins:
     - size
     - skip
     - slackevents
+    - testfreeze
```

Comment:
this is a chicken and egg problem. note that the checkconfig job uses a static image so it won't understand testfreeze plugin until this PR is merged (which would trigger a rebuild of checkconfig image, then there will be an automated process to update checkconfig job)